app.server.js

```diff
@@ -13,6 +13,8 @@ const HOST = "0.0.0.0";
 const GOOGLE_CREDENTIAL =
   process.env.GOOGLE_CREDENTIAL ||
   "990846956506-bfhbjsu4nl5mvlkngr3tsmfcek24e8t8.apps.googleusercontent.com";
+const ENFORCE_SSL = process.env.ENFORCE_SSL || 'true';
```

Comment:
will broken in development environment 🤔

Reply:
yesss I'm so sorry for my mistake. I committed my local test :/